packages/opentelemetry-propagation-utils/src/pubsub-propagation.ts

```diff
@@ -22,13 +22,13 @@ import {
   context,
   trace,
   diag,
-  SpanAttributes,
-} from '@opentelemetry/api';
+  Attributes,
```

Comment:
`Attributes` was added to `@opentelemetry/api` in version `1.1.0` [here](https://github.com/open-telemetry/opentelemetry-js-api/releases/tag/v1.1.0).
The current dependency is on `^1.0.0` which is not compatible. We should either update the dependency or revert to use `SpanAttributes`

Reply:
🤦 , sorry about that. I changed this because my tooling was saying SpanAttributes were deprecated.